keyboards/ekow/akira/keymaps/via/rules.mk

```
@@ -0,0 +1 @@
+ENABLE_VIA = yes
```

Either add a `keymap.c` to make this a vailid keymap or remove this file.
```suggestion
VIA_ENABLE = yes
```